packages/common/package.json

```diff
@@ -95,7 +95,7 @@
     "eslint-plugin-mocha": "10.4.3",
     "mocha": "10.4.0",
     "mocha-junit-reporter": "2.2.1",
-    "playwright": "1.40.1",
+    "playwright": "1.44.0",
```

Comment:
did this get updated in other places?

Reply:
I notice its a different version in packages/agent/package.json

Reply:
I did not update agent related packages because they are not part of the dependabot check.